app/interactors/datapass_webhook/schedule_authorization_request_emails.rb

```diff
@@ -15,7 +13,8 @@ def schedule_email(email_config)
     ScheduleAuthorizationRequestMailjetEmailJob.set(wait_until: extract_when_time(email_config['when'])).perform_later(
       context.authorization_request.id,
       context.authorization_request.status,
-      build_mailjet_attributes(email_config)
+      build_mailjet_attributes(email_config),
+      context.authorization_request.api
```

Comment:
j'hésite à clean en passant juste `authorization_request` plutôt que disséquer en 3 params ici, WDYT? Worth it?

Reply:
Non, le status passé en paramètre est voulu (sinon on utiliserai l'id pour récupérer le modèle et son status). En lisant le code en effet on vérifie si le status a changé dans le futur: https://github.com/etalab/admin_api_entreprise/blob/develop/app/jobs/schedule_authorization_request_mailjet_email_job.rb#L12

Par contre l'api y'a 0 raison que ça change, faut flip ce param